lib/util/state.js

```diff
@@ -118,6 +118,18 @@ export const getActiveItineraries = createSelector(
   }
 )
 
+/**
+ * If there are no itineraries, generate an error
+ * so we're not left with an empty results container.
+ */
+export const createNoItinerariesErr = (state, errors, pending) => {
+  const itineraries = getActiveItineraries(state)
+  if (errors && itineraries.length === 0 && !pending) {
```

Comment:
Clever fix with the pending! But isn't pending a number? Should it be `== 0` instead just for clarity? I could be wrong here

Reply:
By the time it gets here, it's a boolean!